doc/build-netbsd.md

```diff
@@ -1,81 +1,145 @@
+
+### 2. Clone Bitcoin Repo
+Now that `git` and all the required dependencies are installed, let's clone the Bitcoin Core repository to a directory. All build scripts and commands will run from this directory.
+``` bash
 git clone https://github.com/bitcoin/bitcoin.git
 ```
 
-See [dependencies.md](dependencies.md) for a complete overview.
+### 3. Install Optional Dependencies
+
+#### Wallet Dependencies
+It is not necessary to build wallet functionality to run bitcoind or the GUI. To enable legacy wallets, you must install `db4`. To enable [descriptor wallets](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md), `sqlite3` is required. Skip `db4` if you intend to *exclusively* use descriptor wallets
+
+###### Legacy Wallet Support
+`db4` is required to enable support for legacy wallets. Skip if you don't intend to use legacy wallets
+
+```bash
+pkgin install db4
+```
 
-### Building BerkeleyDB
+###### Descriptor Wallet Support
+
+`sqlite3` is required to enable support for descriptor wallets. Skip if you don't intend to use descriptor wallets.
+``` bash
+pkgin install sqlite3
+```
+---
 
-BerkeleyDB is only necessary for the wallet functionality. To skip this, pass
-`--disable-wallet` to `./configure` and skip to the next section.
+#### GUI Dependencies
+###### Qt5
 
-It is recommended to use Berkeley DB 4.8. You cannot use the BerkeleyDB library
-from ports, for the same reason as boost above (g++/libstd++ incompatibility).
-If you have to build it yourself, you can use [the installation script included
-in contrib/](/contrib/install_db4.sh) like so:
+Bitcoin Core includes a GUI built with the cross-platform Qt Framework. To compile the GUI, we need to install `qt5`. Skip if you don't intend to use the GUI.
+```bash
+pkgin install qt5
+```
+###### qrencode
 
+The GUI can encode addresses in a QR Code. To build in QR support for the GUI, install `qrencode`. Skip if not using the GUI or don't want QR code functionality.
 ```bash
-./contrib/install_db4.sh `pwd`
+pkgin install qrencode
 ```
+---
 
-from the root of the repository. Then set `BDB_PREFIX` for the next section:
+#### Test Suite Dependencies
+There is an included test suite that is useful for testing code changes when developing.
+To run the test suite (recommended), you will need to have Python 3 installed:
 
 ```bash
-export BDB_PREFIX="$PWD/db4"
+pkgin install python37
 ```
+---
 
 ### Building Bitcoin Core
 
 **Important**: Use `gmake` (the non-GNU `make` will exit with an error).
 
-With wallet:
+
+### 1. Configuration
+
+There are many ways to configure Bitcoin Core, here are a few common examples:
+##### Wallet (BDB + SQlite) Support, No GUI:
+This explicitly enables legacy wallet support and disables the GUI. An error will be thrown if `db4` is not installed. If `sqlite3` is installed, then descriptor wallet support will also be built.
 ```bash
 ./autogen.sh
-./configure --with-gui=no CPPFLAGS="-I/usr/pkg/include" \
+./configure --with-gui=no \
```

Comment:
While running ./configure ...
config.log is marked as a directory...
I am not sure if this is a result of copy and pasting the multiline command into the terminal...
Or if it is a bug... going to rebuild the image from scratch and run again...

![Screen Shot 2020-12-13 at 11 58 46 PM](https://user-images.githubusercontent.com/152159/102042224-4d3efc80-3d9f-11eb-8c6f-22e74aa9568c.png)